### ISSUE TITLE
Readiness Check while upgrade to 2.0.1 from 2.0.0 fails

### ISSUE THREAD

**Issue by ideadx** [magento/magento2#3069]
<img width="1436" alt="screen shot 2016-01-22 at 10 59 56 am" src="https://cloud.githubusercontent.com/assets/2369963/12499104/8817b3e4-c0f7-11e5-85ff-7fbb86faa94c.png">
![Uploading Screen Shot 2016-01-22 at 11.00.12 AM.png…]()

**Comment by ideadx** [comment]
<img width="1412" alt="screen shot 2016-01-22 at 11 00 12 am" src="https://cloud.githubusercontent.com/assets/2369963/12499110/9888a378-c0f7-11e5-867a-806c5dfd45a8.png">

**Comment by ideadx** [comment]
What does it even means? 
Removing following files :
/mangeto2/var/.update_cronjob_status
/mangeto2/var/.setup_cronjob_status
makes no difference as it is shown here: http://devdocs.magento.com/guides/v2.0/comp-mgr/trouble/cman/component-depend.html

**Comment by mazhalai** [comment]
@ideadx anything in `var/.update_status.log`?

**Comment by ark99** [comment]
@ideadx, please see if browser developer mode would show extra information in the server response that would help identify the cause?

Thanks!

**Comment by ideadx** [comment]
```
An error occurred during execution; please try again later.

Additional information:

ErrorException

File:
/var/www/html/setup/src/Magento/Setup/Controller/DependencyCheck.php:73
Message:
Invalid argument supplied for foreach()
Stack trace:
#0 /var/www/html/setup/src/Magento/Setup/Controller/DependencyCheck.php(73): Composer\Util\ErrorHandler::handle(2, 'Invalid argumen...', '/var/www/html/s...', 73, Array)
#1 /var/www/html/vendor/zendframework/zend-mvc/src/Controller/AbstractActionController.php(82): Magento\Setup\Controller\DependencyCheck->componentDependencyAction()
#2 [internal function]: Zend\Mvc\Controller\AbstractActionController->onDispatch(Object(Zend\Mvc\MvcEvent))
#3 /var/www/html/vendor/zendframework/zend-eventmanager/src/EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#4 /var/www/html/vendor/zendframework/zend-eventmanager/src/EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#5 /var/www/html/vendor/zendframework/zend-mvc/src/Controller/AbstractController.php(118): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#6 /var/www/html/vendor/zendframework/zend-mvc/src/DispatchListener.php(93): Zend\Mvc\Controller\AbstractController->dispatch(Object(Zend\Http\PhpEnvironment\Request), Object(Zend\Http\PhpEnvironment\Response))
#7 [internal function]: Zend\Mvc\DispatchListener->onDispatch(Object(Zend\Mvc\MvcEvent))
#8 /var/www/html/vendor/zendframework/zend-eventmanager/src/EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#9 /var/www/html/vendor/zendframework/zend-eventmanager/src/EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#10 /var/www/html/vendor/zendframework/zend-mvc/src/Application.php(314): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#11 /var/www/html/setup/index.php(31): Zend\Mvc\Application->run()
#12 {main}
```

**Comment by ideadx** [comment]
@ark99 @mazhalai

**Comment by ideadx** [comment]
url that its trying to access:
/setup/index.php/dependency-check/component-dependency

**Comment by mazhalai** [comment]
@ideadx a couple of update related issues have been fixed in 2.0.2, have you had a chance to look at this [tech bulletin](http://devdocs.magento.com/guides/v2.0/release-notes/tech_bull_201-upgrade.html)

**Comment by mazhalai** [comment]
Closing since reporter is non-responsive
